README.md

```diff
@@ -1 +1,71 @@
-# voxels-tutorial
\ No newline at end of file
+# MetaAds Voxel's Scene
+
+## Introduction
+This document is an instruction on how to create a scene for Voxel.
+## 1. Opening the MetaAds website
+* Connect
+* Click on Landowner
+* Click on My Ad Spaces
+## 2. Going to Voxels
+![Image](./media/1.png)
+* Click on Add Ad Space
+* Choose Voxel metaverse
+* Click on Download
+    * Save file on PC
+* Click on To Voxels button
+## 3. Connecting with MetaMask
+![Image](./media/2.png)
+* Click on Sign In
+* Click on MetaMask
+	* Click on Next
+	* Click on Connect
+	* Click on Sign
```

Comment:
Leave the step "Sign In Voxels" only, because user can auth with other wallet

Reply:
done